core/platform/lf_rp2040_support.c

```diff
@@ -203,6 +198,16 @@ int lf_enable_interrupts_nested() {
     return 0;
 }
 
+#if defined(LF_SINGLE_THREADED)
+/**
+ * The single thread RP2040 platform support treats second core
+ * routines similar to external interrupt routine threads.
+ * 
+ * Second core activity is disabled at the same times as
+ * when interrupts are disabled. 
```

Comment:
What is this comment associated with?  Should this be a file-level comment at the top of the file?